domains/ibalh0.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Ibalh0",
+           "email": "",
+           "discord": "1148535309160677376",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Q8OM23V_DPmsBSJrSdkYq4oK3ZzqYOmFumlIfDLjj3xDYnFWSzks4Hvwm_SiplJlPzqTgfsw4FqG51CbcBcuTlUzkhvoK3hyNY8CxZ8Np8_c35JPVo2v_8UkH9H1_qarNYAgquJqk08wGGc32xaaZrpJeN1jO8pCUhL6uc_r2xaJyvMMkO2w8jwu3iqCJpTJH5UH3-XpwJRUT_uzE1RK3rL9laab5TaLR6QIPFgNW5F71bN4Anj3HIlCbG5ee9mJQowSaS5YAMb4fttTYkSNT-1PR3ekKWMieg1z_7K7SbAn4vG3cE0AQj_fL06l6aO_hTWHi5M54WHUbsVj7fjsTQ.nYgVmf3xj0MTFDqn7XA-oA.BOs8XDiRPkOQ4yx9WzNlFJne3kyQPsec2y6g-LWvpvrq4xg9erYPK78xJtJlpyNwjqkxZf3dZW73bKW50kWpWUJ2UC67OWiMTYxXoKPrcs0.fWo46QmZvvKiLirmPoy7kA"
+        },
+    
+        "record": {
+            "CNAME": "https://ibalh0.github.io/ibalh0-portfolio/"
```

Comment:
`CNAME` record must contain a domain only, it cannot contain "`https://`" and "`/`".
You must change it to `ibalh0.github.io`.